proto/cosmos/gov/v1/gov.proto

```diff
@@ -243,4 +243,11 @@ message Params {
 
   //  Minimum expedited deposit for a proposal to enter voting period.
   repeated cosmos.base.v1beta1.Coin expedited_min_deposit = 12 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+
+  // burn deposits if a proposal does not meet qourum
+  bool burn_vote_quorum = 13;
+  // burn deposits if the proposal does not get enough deposits
+  bool burn_proposal_deposit = 14;
```

Comment:
This param name might mislead the dev, as they might get it as burning the proposal deposits in case. wdyt?

Reply:
updated the comment. do you have a suggestion for the name?

Reply:
appended prevote, do you think that is better?

Reply:
> do you think that is better?

much better than the previous.